### ISSUE TITLE
Adding topographical map of Mecklenburg-Vorpommern

### ISSUE THREAD

**Issue by chribre** [osmlab/editor-layer-index#606]
(https://github.com/osmlab/editor-layer-index/files/2687409/AW_.Anfrage.fur.Nutzung.der.WMS.Dienste.im.Rahmen.von.Open.Street.Map.pdf)
PLease add the following WMS-Service I hope I did everything as it should be done. There are more Services to add and I will probably add some more requests from time to time (or should I do all in one?).

## URL of the imagery
JOSM-generated (with all scales; without metadata):

http://www.geodaten-mv.de/dienste/gdimv_dtk?language=ger&FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=mv_dtk10,mv_dtk100,mv_dtk25,mv_dtk50,mv_uek1000&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}

original:
https://www.geodaten-mv.de/dienste/gdimv_dtk?REQUEST=GetCapabilities&SERVICE=WMS

## Two letter country code
DE

## Desired layer name
M-V Topografische Kartenwerke (geodaten-mv)

## License
© GeoBasis-DE/M-V 2018

## Is the license compatible with OSM?
yes (asked for permission)

Permission:
(https://github.com/osmlab/editor-layer-index/files/2687409/AW_.Anfrage.fur.Nutzung.der.WMS.Dienste.im.Rahmen.von.Open.Street.Map.pdf)

## Attribution (text, URL, required?)
-

## Icon
(https://user-images.githubusercontent.com/44900699/50108511-b84bac00-0235-11e9-8901-fb769ffdac2c.png)

## Bounding polygon
https://github.com/osmlab/editor-layer-index/files/2686151/MVboundary.zip

**Comment by grischard** [comment]
Hello!

Can you please attach the permission you've received?

**Comment by chribre** [comment]
> 
> 
> Hello!
> 
> Can you please attach the permission you've received?

Hi, of course it's under "Permission" now

**Comment by grischard** [comment]
https://www.geoportal-mv.de/portal/Geodatenviewer/Datenuebersicht also looks like it would have a few other interesting layers?

The email from Mr Wehden doesn't mention attribution; do you think there's anything we need to do there?

**Comment by chribre** [comment]
Hey,
there are other layers that could be added but I just started to mention two layers becaus I've not been sure how this works here and how many people are involved. It's possible that I switched attribution and license. I think the attribution is "© GeoBasis-DE/M-V 2018" and the license is called AGNB and found here:
(https://github.com/osmlab/editor-layer-index/files/2690900/AfGVK_AGNB.pdf)

I'm not sure if I should post all usable layer at one or add them from time to tim. It would be great if you could help me out here.

**Comment by grischard** [comment]
All at once makes it easier for me ;)

So there are two specific Meckpomm documents regulating the use of geodata, the AGNB which you linked and [Entgelte Geobasisdaten LAiV](https://www.laiv-mv.de/static/LAIV/Abt3.Geoinformation/Dateien/Entgelte_Geobasisdaten_LAiV.pdf) which Mr Wehden cites.

7.3 AGNB says:

> Der Lizenznehmer ist verpflichtet, bei jeder öffentlichen Wiedergabe, Verbreitung oder Präsentation der Geodaten sowie bei jeder Veröffentlichung oder externen Nutzung einer Bearbeitung oder Umgestaltung einen deutlich sichtbaren Quellenvermerk anzubringen

Can you please check with Mr Wehden if they're happy being added to https://www.openstreetmap.org/copyright to satisfy this?

If they say no at first, you can explain that showing every source on every map view is impossible with more than 200000 different sources, and that requiring this would restrict the free reuse of their data.

**Comment by chribre** [comment]
Hey, 
I hope you got me right that I just wanted to add the WMS-Services as layer for remote mapping/checking in the editor it shouldn't be used for data-transfer. If I would ask Mr. Wehden if he accepted the mentioning on the copyright page it would probably appear to him that osm wants to use the geodaten-service as a data-provider because all of the mentioned states offered their ones somehow. And that is not covered by the license and he probably would decline.

Isn't there any possibility to put the attribution maybe into the title of the layer? I think it would cover the "sichtbarer Quellenvermerk". Besides, if you use the wms-service the scene is watermarked. I would guess even that does the job but to be sure it would be nice to mention the source somewhere else. 

Sorry if you have trouble with the way I give you the informations. I will try to help where I can and make the work as easy as possible for you. One way to check if the way we use the services is ok could be that you implement one wms-service and I show it to Frank Wehden for 'verification'.
If he accepts I'll prepare all other helpfull services for you. 
(https://user-images.githubusercontent.com/44900699/50247521-5d9e8580-03d8-11e9-9a6e-953d324d9a85.png)

**Comment by grischard** [comment]
Yes, I have understood you, but I know that this was a blocker in Rheinland-Pfalz.

**Comment by chribre** [comment]
The aerial imagery of berlin also has no attribution in the copyright page but on the right/left buttom in the editor. Couldn't you imagine to do the same with geoportal-stuff instead of the title or copyright-page? I think if I would ask Wehden to do the attribution at the copyright-page it also would be a blocker in MV. Sorry again I just try to keep the possibility of using the services. It's not usual for MV government to be that openminded  ;)
example of the Berlin aerial imagery attribution (JOSM):
(https://user-images.githubusercontent.com/44900699/50285806-9a609000-045d-11e9-95dc-235de9d6f74d.png)

**Comment by chribre** [comment]
Ok Berlin is at the contributors page and it says:
>Die Berliner Senatsverwaltung für Stadtentwicklung und Umwelt (SenStadtUm) stellt seit dem 1. Oktober 2013 Geodaten unter an CC-BY angelehnten Nutzungsbedingungen bereit. Die in § 2 geforderte erkennbare und in optischem Zusammenhang Einbindung der Quellenverweise wird lt. SenStadtUm nur gefordert, wenn die Geodaten einen überwiegenden Anteil im Folgewerk haben. Wenn die veränderten Geodaten nur einen untergeordneten Anteil an dem Folgeprodukt haben, dann kann der Quellenvermerk für abgeleitete Werke auch an einer anderen Stelle (also hier) angebracht werden.
Der Quellenvermerk lautet: Geoportal Berlin / [Titel des Datensatzes] und soll in das source-Tag der Changesets aufgenommen werden.
Weitere Hinweise zur Nutzung des Geoportals Berlin in OSM. 

But beside this there is also an attribution in the editor. I think this would be the perfect solution. If you think it's possible to do it for MV similar to berlin I could ask for the text at the Contributors-page. 

regards
Christian
